extensions/amp-story/0.1/amp-story.js

```diff
@@ -129,6 +137,25 @@ export class AmpStory extends AMP.BaseElement {
 
     /** @private {?./amp-story-page.AmpStoryPage} */
     this.activePage_ = null;
+
+    /** @private @const */
+    this.desktopMedia_ = this.win.matchMedia(
+        `(min-width: ${DESKTOP_THRESHOLD}px)`);
+
+    /** @private @const */
+    this.background_ = new AmpStoryBackground(this.element);
```

Comment:
Can you add a TODO to lazily construct this the first time the user enters desktop mode?

Reply:
Absolutely. It's a low-effort change, I can add it as part of this PR if that's ok